wiki.moinmoin

```diff
@@ -352,17 +352,16 @@ TODO-A: - Team is already subscribed to the package
 TODO-B: - Team is not yet, but will subscribe to the package before promotion
 
 RULE: - Responsibilities implied by static builds promoted to main, which is
-RULE:   not a recommended but a common case with golang packages.
-RULE:   - the security team will track CVEs for all golang packages in main
+RULE:   not a recommended but a common case with golang and rust packages.
+RULE:   - the security team will track CVEs for all vendored packages in main
```

Comment:
```suggestion
RULE:   - the security team will track CVEs for all vendored/embedded sources in main
```
This is not entirely clear IMO. "vendored packages" are meant to be the sources embedded into other packages, right?